tests/scripts/thread-cert/border_router/test_publish_meshcop_service.py

```diff
@@ -111,7 +111,8 @@ def test(self):
         self.check_meshcop_e_service(host, True)
 
         # deactivate ePSKc mode
-        br1.deactivate_ephemeral_key_mode()
+        retain_active_session = False
```

Comment:
also test the case when `retain_active_session = True` ?

Reply:
Add another pair of activate/deactivate for different `retain_active_session` options. Thanks~